ci/functional/junit_list_unsuccessful

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python3
```

Comment:
Recommended, to support python venv, etc.
```suggestion
#!/usr/bin/env python3
```

Reply:
Done.